src/Data/notifyOnConnectionTimeout.tsx

```diff
@@ -0,0 +1,54 @@
+import { currentLanguage } from 'scrivito'
```

Comment:
* [ ] FYI Regarding the location of the module: The only caller is  `CurrentUserDataItem`. There should be no other caller (otherwise we'd show multiple toasts). Thus I think the module should live in `src/Data/CurrentUser/`.